cmd/commands.go

```diff
@@ -242,6 +242,21 @@ provisioning stages.`,
 		}(),
 	})
 
+	RegisterCommand(Command{
+		Name:  "fmt",
+		Func:  cmdFormatConfig,
+		Usage: "[--formatter <name>] [--write] [<path>]",
+		Short: "Formats a configuration file",
+		Long: `
+Formats the configuration file and prints the result on standard output.`,
```

Comment:
This doc text will need some expansion before the final merge (take cues from some of the other commands).

Reply:
Yeah! I'll add more to it